internal/webhook/cleanup.go

```diff
@@ -0,0 +1,113 @@
+package webhook
+
+import (
+	"context"
+
+	kadmissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	kbatchv1 "k8s.io/api/batch/v1"
+	kcorev1 "k8s.io/api/core/v1"
+	kmetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// CleanupResources removes the mutating and validating webhook resources.
+func CleanupResources(ctx context.Context, client kctrlclient.Client) []error {
```

Comment:
is there any unit/integration test for this method?

Reply:
Done